OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -21,6 +20,18 @@ namespace OpenRA.Mods.Common.Activities
 {
 	public class FindResources : Activity
 	{
+		ResourceLayer resLayer;
+		HarvesterInfo harvInfo;
+		ResourceClaimLayer territory;
+		Mobile mobile;
+		Harvester harv;
+		MobileInfo mobileInfo;
+		IPathFinder pathFinder;
```

Comment:
They are assigned later in Tick (see comment on ```loadVars``` below), so that's not possible in this case.